guake/boxes.py

```diff
@@ -381,9 +382,8 @@ def set_search(self, widget):
 
             # Set search regex on term
             self.searchstring = text
-            self.searchre = GLib.Regex(text, 0, 0)
-            term.search_set_gregex(self.searchre, 0)
-
+            self.searchre = Vte.Regex.new_for_search(text, -1, Vte.REGEX_FLAGS_DEFAULT | PCRE2_MULTILINE)
```

Comment:
This is the only line that needs to be changed to pass CI, on my branch I changed it to:

```
            self.searchre = Vte.Regex.new_for_search(
                text, -1, Vte.REGEX_FLAGS_DEFAULT | PCRE2_MULTILINE
            )
```

Reply:
I just went and did it myself.